gazebo/gazebo8/libgazebo8/Dockerfile

```diff
@@ -1,8 +1,14 @@
 # This is an auto generated Dockerfile for gazebo:libgazebo8
 # generated from templates/docker_images/create_gzclient_image.Dockerfile.em
-# generated on 2017-01-26 20:10:11 +0000
+# generated on 2017-01-26 20:31:00 +0000
 FROM gazebo:gzserver8
 MAINTAINER Jose Luis Rivero jrivero@osrfoundation.org
```

Comment:
I'm also thinking of updating the maintainer line:
https://github.com/osrf/docker_templates/issues/11
So I may need to update the template, but its a matter for later.